sc-memory/tests/scs/units/test_scs_common.cpp

```diff
@@ -288,6 +294,36 @@ TEST(scs_common, edges)
   }
 }
 
+TEST(scs_common, reversed_edges)
+{
+  std::string const data =
```

Comment:
This test can be ivalid. Recheck it, please

Reply:
It works correctly